src/backend/CommandBuffer.cpp

```diff
@@ -70,6 +70,16 @@ namespace backend {
             return true;
         }
 
+        bool ValidateBufferOffset(CommandBufferBuilder* builder, TextureBase* texture, const BufferCopyLocation& location) {
+            uint32_t texelSize = static_cast<uint32_t>(TextureFormatPixelSize(texture->GetFormat()));
```

Comment:
An RGB8Unorm format would only has 3 bytes of data, but we don't want to ask to align the buffer to 3. I assume the backing APIs will actually enforce an alignment of 4 bytes, but we'll have to investigate more eventually.